qemu/tests/cfg/multi_disk_random_hotplug.cfg

```diff
@@ -51,9 +51,11 @@
     variants:
         - all_types:
             stg_params = "fmt:virtio,virtio_scsi,lsi_scsi,usb2"
-            pseries:
-                stg_params = "fmt:virtio,virtio_scsi,usb2"
             usbs += " ehci"
+            pseries:
+                stg_params = "fmt:virtio,virtio_scsi,usb3"
```

Comment:
Does RHEL7 or old qemu versions also apply to this rule?

Reply:
Upstream qemu still have ehci controller for power platform，downstream qemu  have no this controller from qemu-kvm-rhev-2.12.0,so add change in internal config and just add usb3 fmt support on py file here.

Reply:
OK, thank you.